platform/windows/display_server_windows.cpp

```diff
@@ -2421,6 +2421,9 @@ bool DisplayServerWindows::window_get_flag(WindowFlags p_flag, WindowID p_window
 		case WINDOW_FLAG_POPUP: {
 			return wd.is_popup;
 		} break;
+		case WINDOW_FLAG_SHARP_CORNERS: {
+			return wd.sharp_corners;
+		} break;
```

Comment:
I would remove the case here to put it at the same relative place that the setter does it. (just after `WINDOW_FLAG_ALWAYS_ON_TOP`)

```suggestion
```